gazel/gazel.go

```diff
@@ -46,14 +46,17 @@ func main() {
 	if err != nil {
 		glog.Fatalf("unable to build vendorer: %v", err)
 	}
+	if err := os.Chdir(v.root); err != nil {
+		glog.Fatalf("cannot chdir into root %q: %v", v.root, err)
+	}
 
 	if err := v.walkVendor(); err != nil {
 		glog.Fatalf("err walking vendor: %v", err)
 	}
 	if err := v.walkRepo(); err != nil {
 		glog.Fatalf("err walking repo: %v", err)
 	}
-	if _, err := v.walkSource(*root); err != nil {
+	if _, err := v.walkSource("."); err != nil {
```

Comment:
it's a recursive function. I could write a wrapper, but it's not clear how much effort it's worth...